oni-lda-c

```diff
@@ -1 +1 @@
-Subproject commit 007ae0cda2df947cc64074c3ca99f8155db5bc3c
+Subproject commit 6cb0e03e7b78e45558a10774a8922b1f948ae8cd
```

Comment:
what on earth does this mean?
how do I know the hash on the right is the correct one?